v1.1/architecture/overview.md

```diff
@@ -55,7 +55,7 @@ Term | Definition
 **Consensus** | When a range receives a write, a quorum of nodes containing replicas of the range acknowledge the write. This means your data is safely stored and a majority of nodes agree on the database's current state, even if some of the nodes are offline.<br/><br/>When a write *doesn't* achieve consensus, forward progress halts to maintain consistency within the cluster.
 **Replication** | Replication involves creating and distributing copies of data, as well as ensuring copies remain consistent. However, there are multiple types of replication: namely, synchronous and asynchronous.<br/><br/>Synchronous replication requires all writes to propagate to a quorum of copies of the data before being considered committed. To ensure consistency with your data, this is the kind of replication CockroachDB uses.<br/><br/>Aysnchronous replication only requires a single node to receive the write to be considered committed; it's propogated to each copy of the data after the fact. This is more or less equivalent to "eventual consistency", which was popularized by NoSQL databases. This method of replication is likely to cause anomalies and loss of data.
 **Transactions** | A set of operations performed on your database that satisfy the requirements of [ACID semantics](https://en.wikipedia.org/wiki/Database_transaction). This is a crucial component for a consistent system to ensure developers can trust the data in their database.
-**Multi-Active Availability** | A consensus-based notion of high availability that lets all nodes in a cluster receive reads and writes. This is similar to active-active replication, but circumvents the possibility of nodes having conflicting information.
+**Multi-Active Availability** | Our consensus-based notion of high availability that lets each node in the cluster handle reads and writes for a subset of the stored data (on a per-Range basis). This is in contrast to active-passive replication, in which the active node receives 100% of request rraffic, and it is also not active-active replication, in which all nodes accept requests but typically can't guarantee that reads are both up-to-date and fast.
```

Comment:
LGTM, but I think @sploiselle should take a look, too. 

One error: `rraffic` > `traffic`. I can fix, though, once Sean chimes in.